Tasks/CopyFilesOverSSHV0/sshhelper.ts

```diff
@@ -73,6 +73,9 @@ export class SshHelper {
     closeConnection() {
         try {
             if (this.sftpClient) {
+                this.sftpClient.on('error', (err) => {
+                    tl.debug('sftpClient: Ignoring error diconnecting: ' + JSON.stringify(err));
```

Comment:
Are you sure we want to serialize as JSON?  I think we just want `+ err` so it will `.toString()` it.